sdk/tests/conformance2/reading/format-r11f-g11f-b10f.html

```diff
@@ -0,0 +1,165 @@
+<!--
+
+/*
+** Copyright (c) 2016 The Khronos Group Inc.
+**
+** Permission is hereby granted, free of charge, to any person obtaining a
+** copy of this software and/or associated documentation files (the
+** "Materials"), to deal in the Materials without restriction, including
+** without limitation the rights to use, copy, modify, merge, publish,
+** distribute, sublicense, and/or sell copies of the Materials, and to
+** permit persons to whom the Materials are furnished to do so, subject to
+** the following conditions:
+**
+** The above copyright notice and this permission notice shall be included
+** in all copies or substantial portions of the Materials.
+**
+** THE MATERIALS ARE PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+** EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+** MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+** IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
+** CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+** TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+** MATERIALS OR THE USE OR OTHER DEALINGS IN THE MATERIALS.
+*/
+
+-->
+
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>Test Format R11F_G11F_B10F</title>
+<link rel="stylesheet" href="../../resources/js-test-style.css"/>
+<script src="../../js/js-test-pre.js"></script>
+<script src="../../js/webgl-test-utils.js"></script>
+</head>
+<body>
+<div id="description"></div>
+<div id="console"></div>
+<canvas id="canvas" width="20" height="20"> </canvas>
+<script>
+"use strict";
+description("This tests format R11F_G11F_B10F works as expected");
+debug("MacOSX driver bug. See https://github.com/KhronosGroup/WebGL/issues/1832");
+
+var wtu = WebGLTestUtils;
+var canvas = document.getElementById("canvas");
+var gl = wtu.create3DContext(canvas, undefined, 2);
+
+if (!gl) {
+  testFailed("context does not exist");
+} else {
+  testPassed("context exists");
+  if (gl.getExtension("EXT_color_buffer_float")) {
+    testPassed("Extension EXT_color_buffer_float is available");
+    testRenderbufferReadback(4, 4);
+    testTextureReadback(4, 4);
+  } else {
+    testPassed("Extension EXT_color_buffer_float is unavailable - this is legal");
+  }
+}
+
+function setupColor(testR, testG, testB, value) {
+  var data = new Float32Array(4);
+  data[0] = testR ? value : 0;
+  data[1] = testG ? value : 0;
+  data[2] = testB ? value : 0;
+  data[3] = 1; // Doesn't really matter for RGB formats.
+  return data;
+}
+
+function clearAndVerifyColor(width, height, testR, testG, testB, value) {
+  var data = setupColor(testR, testG, testB, value);
+  // TODO(zmo): are we too tolerant here?
+  var tol = [36, 36, 36];
```

Comment:
I try to fix the bug at https://github.com/KhronosGroup/WebGL/pull/1850. Could you have a look? @zhenyao , @kenrussell .